info/guide/guide-chapter-3.md

```diff
@@ -83,7 +83,7 @@ Kotlin allows several top-level declaration types: classes, objects, interfaces,
 When declaring more than one class or zero classes (e.g. only functions), as per rule [2.2.1](#r2.2.1), you should document the whole file in the header KDoc.
 When declaring top-level structures, keep the following order:
 1. Top-level constants and properties (following same order as properties inside a class: `const val`,`val`, `lateinit var`, `var`)
-2. typealiases (grouped by their visibility modifiers)
+2. Type-aliases (grouped by their visibility modifiers)
```

Comment:
In the Kotlin documentation, they're called _type aliases_: https://kotlinlang.org/docs/type-aliases.html

```patch
--- Type-aliases
+++ Type aliases
```